.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Are we sure that specifying the branch here is correct? I don't see CI on this PR running. Here's how we did this on Swift Inspector FWIW: https://github.com/fdiaz/SwiftInspector/blob/53aec4d6fa12ba0801ca97b1368eeae60a940a59/.github/workflows/ci.yml#L3

Reply:
This is the same configuration we use for Lottie and Epoxy: https://github.com/airbnb/lottie-ios/blob/master/.github/workflows/main.yml#L3

Will try what you have for SwiftInspector though, thanks for the suggestion.

Reply:
Worked correctly after I merged this PR: https://github.com/airbnb/swift/pull/168#discussion_r925104433